File: hw/CEDENOR/U2/HW15MongoShell&Code/HW15MongoShell&Code/mongoDB.py
from pymongo import MongoClient

client = MongoClient("mongodb+srv://nncedeno1:beaTLES123@cluster0.tkhbv81.mongodb.net/")

db = client['OOP']

coleccion = db['students']

documento = {
    "id": "L00394185",
    "name": "Nahomi Cedeño",
    "male": True,
    "siblings": 4
}

resultado = coleccion.insert_one(documento)

print(f"Documento agregado con ID: {resultado.inserted_id}")

File: hw/ccedeno/u2/HW15MongoShell&Code/MongoJs/mongoDB.js
const { MongoClient } = require('mongodb');

async function main() {
    const uri = "mongodb+srv://cedeno:cedeno@cluster0.arabu7a.mongodb.net/";

    const client = new MongoClient(uri);

    try {
        await client.connect();

        const db = client.db('opp');
        const coleccion = db.collection('student');

        const documento = {
            name: "Andres",
            male: true,
            siblings: 2
        };

        const resultado = await coleccion.insertOne(documento);
        console.log(`Se agregó: ${resultado.insertedId}`);
    } catch (e) {
        console.error(e);
    } finally {
        await client.close();
    }
}

main().catch(console.error);